mypy_boto3_builder/stubs_static/types-aiobotocore/httpsession.pyi

```diff
@@ -21,8 +21,8 @@ class AIOHTTPSession:
     async def __aenter__(self: _R) -> _R: ...
     async def __aexit__(
         self,
-        exc_type: Optional[Type[Exception]],
-        exc_val: Optional[Exception],
+        exc_type: Optional[Type[BaseException]],
+        exc_val: Optional[BaseException],
```

Comment:
Should be moved to https://github.com/youtype/types-aiobotocore/blob/main/aiobotocore-stubs/httpsession.pyi